### PR TITLE
Ajoute la case F2FA dans l'impôt sur le revenu

### PR DESCRIPTION
**Pull request by edarin**
Supersedes #537 (to read the discussion)

### CODE REVIEW

File: openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py [hunk 1]
```diff
@@ -724,14 +725,17 @@ def function_20130101_20151231(self, simulation, period):
...
         DEF = deficit_rcm
-        return period, max_(TOT1 + TOT2 + TOT3 - DEF, 0)
+        return period, max_(TOT1 + TOT2 + TOT3 + TOT4 - DEF, 0)
 
 
 class rfr_rvcm(Variable):
     column = FloatCol
     entity_class = FoyersFiscaux
-    label = u"rfr_rvcm"
+    label = u"Revenu fiscal de référence - Abattement - Capitaux"
```
Comment (benjello) [inline openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py]: Improve label: revenus des valeurs et capitaux mobiliers

File: openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py [hunk 2]
```diff
@@ -1828,6 +1833,23 @@ def function_20080101_20121231(self, simulation, period):
             - prelevement_liberatoire.assvie * f2dh
         return period, out
 
+    @dated_function(start = date(2013, 1, 1))
+    def function_20130101(self, simulation, period):
+        '''
+        Prelèvement libératoire sur les revenus du capital
```
Comment (benjello) [inline openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py]: Prélèvement

**Comment by edarin** [comment]
rev_cat_rvcm : include all rev_cap_lib except the cerfa box 2dh.
 why ? Is it just forgotten ?

**Comment by benjello** [comment]
Do not know. I would look at the fiche de calcul (or better the code M of the claculette)

**Comment by edarin** [comment]
my great problem is that there is no -at least what I know about- a precise definition of rvcm neither in the feuille de calcul nore in the Mcode.

They don't think with this concept - so I don't know where to check ...

**Comment by edarin** [comment]
my idea would then be : 
- to include 2dh in `rev_cat_rvcm` so that`rev_cat_rvcm` entails perfectly  `rev_cap_lib`
- and then to exclude `rev_cap_lib` whenever the both variable are called

**Comment by benjello** [comment]
Sounds reasonable.
But do a last check. The amount in 2DH mighti already be included in other variables. 2DH is only used to deduct some already paid CSG if I record correctly.

**Comment by edarin** [comment]
If I traced it back correctly it is more for a crédit d'impôt and it's only included in `prlire` which has no link to our fiel in concern

**Comment by benjello** [comment]
prlire is a restitution of 7.5 % and it looks like a reimbursment of prelevement liberatoire on assurance-vie.
So it might be the part of the rev_cap_lib that can be reimbursed. 
You could do a last check by looking at INES ...
Otherwise do not include it (keep it as it is) and open another issue to be able to push your work.

**Comment by edarin** [comment]
I completely agree with what you say about what is prlire.

Then I can just go to step 2 and use only rev_cat_rvcm instead of rev_cap_lib without changing the first one. And add in formulas of rev_coll and bouclierfiscal manualy this cerfa box ?

**Comment by benjello** [comment]
When both rev_cat_rvcm and rev_cap_lib are present you should prefer rev_cat_rvcm which is more general. if the formula is clearly about prelevement liberatoire then use rev_cap_lib.
hope it helps

**Comment by edarin** [comment]
I think that now we have finish with this case and I export issues I have met here : #544

**Comment by ClaireLeroyIPP** [comment]
Pour info, dans la PR #1034 j'ai introduit la case F2FA dans le calcul de l'impôt (elle entre dans les revenus du capital imposé au prélèvement libératoire ). Je ne sais pas si cela correspond à ce que cette PR souhaitait faire ?

**Comment by benjello** [comment]
Oui @claireleroy je pense que c'est bien cela. Pourrais-tu rapidement vérifier à l'oeil (onglet changes) s'il n'y a pas d'élément nouveau par rapport à ce que tu as fait (sachant que ce code date un peu) ?
Je pense que non et cela nous permettrait de fermer cette PR.

**Comment by claireleroy** [comment]
@benjello Après vérification je pense que c'est bon !

**Comment by benjello** [comment]
Merci @claireleroy, je ferme.

**Pull request closed without merging**